android/app/src/main/java/org/smartregister/fhircore/activity/BaseSimpleActivity.kt

```diff
@@ -172,7 +172,7 @@ abstract class BaseSimpleActivity :
     }
   }
 
-  private fun setMenuCounter(@IdRes itemId: Int, count: Int) {
+  fun setMenuCounter(@IdRes itemId: Int, count: Int) {
```

Comment:
is this (and visibility changes below) only for testing purpose? If so, I do not think we should do this, here's an article about how to use reflection to test private funs

https://medium.com/mindorks/how-to-unit-test-private-methods-in-java-and-kotlin-d3cae49dccd

Reply:
logic updated using with reflection api. Thanks